pysat/instruments/icon_fuv.py

```diff
@@ -126,9 +124,12 @@ def default(inst):
 
     """
 
-    import pysat.instruments.icon_ivm as icivm
-    inst.tag = 'level_2'
-    icivm.remove_icon_names(inst, target='ICON_L2_FUV_Daytime_ON2_')
+    # Use datetime instead of timestamp for Epoch
+    inst.data['Epoch'] = pds.to_datetime([dt.datetime.utcfromtimestamp(x / 1000)
+                                          for x in inst.data['Epoch']])
+    target = {'day': 'ICON_L24_',
+              'night': 'ICON_L25_'}
+    mm_gen.remove_leading_text(inst, target=target[inst.tag])
```

Comment:
Same note as MIGHTI.  Would probably be cleanest to have all replacements in a routine in methods that are called here.  Could also have a 'revert_variable_names' routine that changes everything back for insane users who like the ICON codes.

Reply:
We could add a keyword to the instrument load method, and then check if that keyword was provided at instantiation in default before invoking remove_preamble. I'm a bit behind at the moment so starting with a comment rather than jump in with code.

Reply:
I like the keyword version.  Can you implement this @rstoneback?  Just pushed the updates to the other review comments.